cms-YYYY-validated-runs/README.md

```diff
@@ -1,3 +1,39 @@
 # cms-YYYY-validated-runs
 
-This directory contains a helper script to prepare CMS validated runs records.
\ No newline at end of file
+This directory contains a helper script to prepare CMS validated runs records.
+
+The python3 script `vali_records.py` can be used to build the records for standard pp data taking and HI-related runs. 
+
+Inputs arguments:
+
+- recid (May 2023, the last used recid was 14210 for 2015 pp)
+- year to be released
+- run era (for pp, give one of the released eras, it is needed to get the collision energy, stored per era)
+- type (pp, pPb, pphiref, PbPb)
+
+Input values:
+
+- 2013 pPb: 14216 2013 HIRun2013 pPb
+- 2013 ppref: 14218 2013 Run2013A pphiref
+- 2015 ppref: 14212 2015 Run2015E pphiref
+- 2015 pp:  14210 2015 Run2015D pp (not used to build the current record, but tested to produce the same files)
+
+
+The script does not require CMSSW release area and it can be run with python3, e.g.:
+
+```
+python3 vali_records.py 14212 2015 Run2015E pphiref > output.json
```

Comment:
We can say `python3 ./code/vali_records.py ...` so that the command would be copy-paste friendly from the same current working directory where this README file lives. (i.e. `cms-YYYY-validated-runs`).

Reply:
Ah yes, that's right, thanks!